containerbuddy/onchange_reload-nfs.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+if [[ `curl -s ${CONSUL}:8500/v1/health/state/passing | grep nfs`  ]]
+then
+  echo "nfs is healthy, mounting uploads directory...."
+  mount -t nfs -v -o nolock,vers=3 nfs:/exports /var/www/html/content/uploads
+else
```

Comment:
This else condition is dangerous. On the one hand, the container would be blocked if we don't unmount it, on the other hand, what we're doing is shifting the failure elsewhere:

- reads will 404
- writes will partition with very poor recoverability options.

If we assume that reads will be cached elsewhere (limiting the problems that will get to this disconnected instance), is there a way to put WP into read-only mode WRT the uploads?

(previously in https://github.com/10up/triton-wordpress/pull/2#discussion_r58460745)